pkg/services/ngalert/api/tooling/definitions/alertmanager_validation_test.go

```diff
@@ -0,0 +1,242 @@
+package definitions
+
+import (
+	"testing"
+
+	"github.com/prometheus/common/model"
+	"github.com/stretchr/testify/require"
+)
+
+func TestValidateRoutes(t *testing.T) {
+	zero := model.Duration(0)
+
+	type testCase struct {
+		desc   string
+		route  Route
+		expMsg string
+	}
+
+	t.Run("valid route", func(t *testing.T) {
+		cases := []testCase{
+			{
+				desc:  "empty",
```

Comment:
I doubt we want the empty route to pass validation long-term, but it does today. I intentionally did not change this in this PR, the goal of this PR is to have no changes on the actual behavior of existing validation.